### DISCUSSION TITLE
登录拦截不生效，出现先进入主页（也触发了后端获取当前用户接口），再重定向回登录页？

### DISCUSSION THREAD

**Discussion by pkaq** [ant-design/ant-design#37097] [Q&A]
关于 #3808 这个issue在v6中有无一种优雅一些的解决方式，目前是在app.tsx中判断是否登录页前面加上了本地是否存在用户token的判断来阻止默认访问 但似乎在登录后再次调用fetchuserinfo的时候会引起 ` Can't perform a React state update on an unmounted component`的问题

**Comment by huanglei1991** [comment]
这个和antd 没啥直接关系 你跑这问没人会回答的。。
